gulp/tests/prod-build/prod-build.setup.js

```diff
@@ -13,5 +13,14 @@ import { filesToMock } from './prod-build.utils';
 
 // Copy the mock files to their destination before testing.
 filesToMock.forEach( ( file ) => {
+	// Check if the file already exists
+	const fileExists = fs.existsSync( file.dest );
+	if ( fileExists ) {
+		// If it does, rename the file adding an -existing suffix
+		const existingFile = file.dest.replace( /(\.[\w\d_-]+)$/i, '-existing$1' );
```

Comment:
This regexp could be simplified like this: `/(\.[\w-]+)$/i`
`\w` matches `[a-zA-Z0-9_]`, we don't need `\d` and `_`.

Reply:
@benoitchantre updated